Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1069,6 +1080,8 @@ def __call__(self, message):
         text = self.encode(message)
         if self.buffering:
             self.messages.append(text)
+            if self.debug_logging:
```

Comment:
isnt better not to push it to messages in the first place in debug mode instead of checking it in `print_log`?

Reply:
I want to leave as is. As we later on write all the appended message to the war room sometimes. There may be issues associated to this behaviour (for example very large entry or something else).